website/docs/routes/university.mdx

```diff
@@ -3853,7 +3853,7 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 ### 信息科学与工程学院 {#zhong-guo-hai-yang-da-xue-xin-xi-ke-xue-yu-gong-cheng-xue-yuan}
 
-<Route author="GeoffreyChen777" example="/ouc/it/0" path="/ouc/it/:type?" paramsDesc={['默认为 `0`']} radar="1">
+<Route author="GeoffreyChen777 3401797899" example="/ouc/it/0" path="/ouc/it/:type?" paramsDesc={['默认为 `0`']} radar="1">
```

Comment:
Please update https://github.com/DIYgod/RSSHub/blob/0e94ae3db2316a848595b788167bf6221d5eb4c1/lib/v2/ouc/maintainer.js#L3
as well.

Reply:
I have updated it.